packages/e2e-tests/tests/console_warp-02.test.ts

```diff
@@ -56,6 +58,10 @@ test("console_warp-02: support pausing, warping, stepping and evaluating console
   await stepOverToLine(page, 21);
   await verifyPausedAtMessage(page, "1 << 7", "terminal-expression");
 
+  // TODO: extra line
+  if (BrowserName == "chromium") {
+    await stepOverToLine(page, 21);
+  }
```

Comment:
cool. but lets just make this a chromium test and stop using the firefox example.